packages/sui-ssr/server/ssr/index.js

```diff
@@ -48,6 +49,15 @@ export default (req, res, next) => {
         'rel="stylesheet"',
         'rel="stylesheet" media="only x" as="style" onload="this.media=\'all\'"'
       )
+      .replace(
+        HEAD_CLOSING_TAG,
+        `
+        <script>
+        /*! loadCSS. [c]2017 Filament Group, Inc. MIT License */
```

Comment:
We might want to extract this to be reused here and with the dynamic rendering.

Reply:
Maybe with the Raw loader. But I am not sure if it works in the server config